File: app/src/main/java/in/ac/bvmengineering/udaan2k19/Adapter/EventManagerAdapter.java
package in.ac.bvmengineering.udaan2k19.Adapter;

import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.util.ArrayList;

import in.ac.bvmengineering.udaan2k19.DataClass.Manager;
import in.ac.bvmengineering.udaan2k19.R;

public class EventManagerAdapter extends RecyclerView.Adapter<EventManagerAdapter.EventManagerViewHolder> {

    private ArrayList<Manager> managers;

    public EventManagerAdapter(ArrayList<Manager> managers) {
        this.managers = managers;
    }

    @NonNull
    @Override
    public EventManagerViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
        return new EventManagerViewHolder(LayoutInflater.from(viewGroup.getContext()).inflate(R.layout.contact_list_item, viewGroup, false));
    }

    @Override
    public void onBindViewHolder(@NonNull EventManagerViewHolder eventManagerViewHolder, int i) {
        Manager manager = managers.get(i);
        eventManagerViewHolder.name.setText(manager.getName());
        eventManagerViewHolder.phone.setText(manager.getPhone());
    }

    @Override
    public int getItemCount() {
        return managers.size();
    }

    class EventManagerViewHolder extends RecyclerView.ViewHolder {
        TextView name, phone;

        EventManagerViewHolder(@NonNull View itemView) {
            super(itemView);
            name = itemView.findViewById(R.id.contact_name);
            phone = itemView.findViewById(R.id.contact_number);
        }
    }
}

File: app/src/main/java/in/ac/bvmengineering/udaan2k19/Adapter/EventManagerAdapter.kt
package `in`.ac.bvmengineering.udaan2k19.Adapter

import `in`.ac.bvmengineering.udaan2k19.DataClass.Manager
import `in`.ac.bvmengineering.udaan2k19.R
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.contact_list_item.view.*

class EventManagerAdapter(private val managers: ArrayList<Manager>) : RecyclerView.Adapter<EventManagerAdapter.EventManagerViewHolder>() {
    override fun onCreateViewHolder(p0: ViewGroup, p1: Int): EventManagerViewHolder {
        return EventManagerViewHolder(LayoutInflater.from(p0.context).inflate(R.layout.contact_list_item, p0, false))
    }

    override fun getItemCount(): Int {
        return managers.size
    }

    override fun onBindViewHolder(p0: EventManagerViewHolder, p1: Int) {
        val manger = managers[p1]
        p0.name.text = manger.name
        p0.phone.text = manger.phone
    }

    class EventManagerViewHolder(view: View) : RecyclerView.ViewHolder(view) {
        val name = view.contact_name!!
        val phone = view.contact_number!!
    }
}